OpenSim/Examples/Plugins/CoupledBushingForceExample/CoupledBushingForce.h

```diff
@@ -41,8 +40,8 @@ namespace OpenSim {
  * where components of the resulting BodyForce are coupled to all any or all 
  * deviations, such that the general stiffness and damping matrices are 6x6.
  *
- * Deviations of a frame1 on body1 and another (frame2) on body2 are represents in
- * termsof x-y-z Euler angle sequence and x,y,z position of frame2 in frame1.
+ * Deviations of a frame1 on frame1 and another (frame2) on body2 are represents
```

Comment:
represented.

This comment is confusing to me; it now seems inconsistent (frame1 on frame1, frame2 on body2).